test/lib/Elastica/Test/QueryTest.php

```diff
@@ -272,6 +272,23 @@ public function testSetQueryToArrayCast()
     /**
      * @group unit
      */
+    public function testNotCloneInnerObjects()
+    {
+        $query = new Query();
+        $termQuery = new Term();
+        $termQuery->setTerm('text', 'value');
+        $query->setQuery($termQuery);
+
+        $anotherQuery = clone $query;
+
+        $termQuery->setTerm('text', 'another value');
+
+        $this->assertEquals($query->toArray(), $anotherQuery->toArray());
```

Comment:
On my mind it is expected now. I don't see any strong or even weak arguments against.

When somebody will need full clone with inner objects, better to make "duplicate" method.